decidim-core/app/views/devise/shared/_links.html.erb

```diff
@@ -0,0 +1,25 @@
+<%- if controller_name != 'sessions' %>
+  <%= link_to t(".sign_in"), new_session_path(resource_name) %><br />
+<% end -%>
+
+<%- if devise_mapping.registerable? && controller_name != 'registrations' %>
```

Comment:
We can probably get rid of all these checks since we know which modules we have enabled and which modules we don't?

Reply:
There's some discussion on others organizations there is the possibility to have users registration disabled (ie https://github.com/AjuntamentdeBarcelona/decidim-barcelona-organizations/) 
cc/ @josanFFiG